Animations/vendor/glad/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# This glad library has been generated using gen.glad.sh
```

Comment:
I've never used this before, I'm just curious if this is the same as using this site or running this repo locally https://glad.dav1d.de ? If it is, is there any particular reason you deleted the glad files that were originally there and replaced them?

Reply:
Basically this is just version 2 of Glad. I only upgraded the library because I had to regenerate the files anyway. I have no idea why the URL is just totally different, but the library author and maintainer is the same.